kubermatic/structure_cluster.go

```diff
@@ -110,6 +102,14 @@ func flattenAWSCloudSpec(in *models.AWSCloudSpec) []interface{} {
 		att["security_group_id"] = in.SecurityGroupID
 	}
 
+	if in.AccessKeyID != "" {
```

Comment:
you moved that code because of some of your tests? It's ok if yes, or that was made on purpose, then what's the reason?

Reply:
I will roll back this code back.

I was editing here cause there was an undesired behavior. Each time `terraform apply` is run, cluster gets updated even if nothing has changed. This is due to local state rewrite which happens before each `terraform apply`. Ideally when state is refreshed, secrets should not be rewritten.